src/helpers/MiscFunctions.cpp

```diff
@@ -161,7 +161,7 @@ std::string getFormat(const char* fmt, ...) {
 
     va_list args;
     va_start(args, fmt);
-    vasprintf(&outputStr, fmt, args);
+    std::ignore = vasprintf(&outputStr, fmt, args);
```

Comment:
I don't believe all of these belong in this PR.

Reply:
> I also added std::ignores to a few unused results to suppress the warnings, I can remove those if you want to.

Alright, reverted. Warnings were annoying me during dev tbh